examples/demo_workflow/demo_workflow/requirements.txt

```diff
@@ -1 +1 @@
-dapr-ext-workflow-dev>=0.0.1rc1.dev
+dapr-ext-workflow-dev>=0.4.1rc1.dev
```

Comment:
Please do not modify this in the future. We want to run examples against older versions for backwards compatibility as well. It's fine here given the SDK is not 1.0 yet.